testing/MLDB-180-basic-join.js

```diff
@@ -258,7 +258,7 @@ for (var i in funcs) {
 // MLDB-1088
 
 expected = [
-   [ "_rowName", "test1.x", "test1.y", "test1.z" ],
+   [ "_rowName", "x", "y", "z" ],
```

Comment:
is this a generalized change? i.e. when I join `a` and `b` the column names are no longer prefixed by `a` and `b` ?